h/views/api_flags.py

```diff
@@ -22,6 +22,23 @@ def create(context, request):
     return HTTPNoContent()
 
 
+@api_config(route_name='api.flags',
+            request_method='GET',
+            link_name='flag.index',
+            description='List a users flagged annotations for review.',
+            effective_principals=security.Authenticated)
```

Comment:
I'd rather stay consistent with the rest of our API and respond with a 404 instead of providing a dummy response.